includes/template-functions.php

```diff
@@ -725,7 +725,31 @@ function sensei_the_question_content() {
 	?>
 
 	<div class="wp-block-sensei-lms-question-answers">
-		<?php Sensei_Question::load_question_template( $question_type ); ?>
+		<?php
+			/**
+			 * Fires before the question answers are displayed inside the answers block.
+			 *
+			 * @hook sensei_quiz_question_answers_inside_before Hook used for printing content inside the question answers section before the template content.
+			 *
+			 * @since $$next-version$$
+			 *
+			 * @param int $question_id The ID of the question.
+			 */
+			do_action( 'sensei_quiz_question_answers_inside_before', $sensei_question_loop['current_question']->ID );
```

Comment:
Hooks need to follow [this format](https://senseip2.wordpress.com/2020/07/06/sensei-lms-hook-docs/) or they won't show up in our hook docs. In particular, the param is missing the curly brackets.

Reply:
Thanks! Updated it here https://github.com/Automattic/sensei/pull/7129/commits/80e18cb367749070fd020b84b286ab86f4371905